admin/server/alerts.go

```diff
@@ -471,6 +471,7 @@ func (s *Server) yamlForManagedAlert(opts *adminv1.AlertOptions, ownerUserID str
 	res.Annotations.AdminOwnerUserID = ownerUserID
 	res.Annotations.AdminManaged = true
 	res.Annotations.AdminNonce = time.Now().Format(time.RFC3339Nano)
+	res.Annotations.DashboardMeasureFilterIndices = opts.DashboardMeasureFilterIndices
```

Comment:
1. Maybe use `MetricsView` as the prefix instead of `Dashboard` (now that we're introducing custom dashboards as well)
2. What will a value for this field look like? A little unclear on the purpose